CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 cmake_minimum_required(VERSION 3.5)
 project(nextpnr CXX C)
 
+# Allow family.cmake add additional dependencies to gui_${family}.
+cmake_policy(SET CMP0079 NEW)
+
+# Enable IPO support.
```

Comment:
Is IPO intended to be part of this patch?

Reply:
I can remove it.  There is also some linking against `libprofiler` in here too.  I can seperate it out.

Reply:
I was planning on adding it.

Reply:
I've split up the commits with clear commit descriptions, and updated the PR description and title to avoid confusion.

Reply:
Thanks!